src/App/screens/Instructor/state/actions/instructorActionTypes.js

```diff
@@ -1,8 +1,11 @@
-export const REQUESTED_INSTRUCTOR = 'REQUESTED_INSTRUCTOR'
-export const RECEIVED_INSTRUCTOR = 'RECEIVED_INSTRUCTOR'
+export const STARTED_FETCH_INSTRUCTOR = 'STARTED_FETCH_INSTRUCTOR'
+export const ENDED_FETCH_INSTRUCTOR = 'ENDED_FETCH_INSTRUCTOR'
 
-export const REQUESTED_INSTRUCTOR_LESSONS = 'REQUESTED_INSTRUCTOR_LESSONS'
-export const RECEIVED_INSTRUCTOR_LESSONS = 'RECEIVED_INSTRUCTOR_LESSONS'
+export const STARTED_FETCH_INSTRUCTOR_LESSONS = 'STARTED_FETCH_INSTRUCTOR_LESSONS'
+export const ENDED_FETCH_INSTRUCTOR_LESSONS = 'ENDED_FETCH_INSTRUCTOR_LESSONS'
 
-export const REQUESTED_ALL_LESSONS = 'REQUESTED_ALL_LESSONS'
-export const RECEIVED_ALL_LESSONS = 'RECEIVED_ALL_LESSONS'
+export const STARTED_FETCH_ALL_LESSONS = 'STARTED_FETCH_ALL_LESSONS'
+export const ENDED_FETCH_ALL_LESSONS = 'ENDED_FETCH_ALL_LESSONS'
+
+export const STARTED_UPDATE_LESSON_STATE = 'STARTED_UPDATE_LESSON_STATE'
+export const ENDED_UPDATE_LESSON_STATE = 'ENDED_UPDATE_LESSON_STATE'
```

Comment:
I made all actions use a start/end prefix so it is easier to follow the logic:
![image](https://cloud.githubusercontent.com/assets/5497885/20736086/92621280-b661-11e6-8b6b-aae6519c8152.png)